cmd/swarmctl/secrets/create.go

```diff
@@ -0,0 +1,44 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/cmd/swarmctl/common"
+	"github.com/spf13/cobra"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create <secret name>",
+	Short: "Create a secret",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("create command takes a unique secret name as an argument, and accepts secret data via stdin")
```

Comment:
I don't want to hold up this PR for this since the docker UX can and will be different, but I don't necessarily agree.

1) `echo blah | docker secret create` would show up in the process list / shell history as well
2) If you have local root access, you can grab the secrets from memory anyway
3) stdin can be messy for the users, especially since we don't display the secrets back to the user. Having a trailing `\n` will be a pretty common problem
4) All the secret stores I've played with so far allow to provide secrets through the command line
5) A manager is not "the enemy". We're not trying to protect secrets against the manager machine itself